ext/dav1d_android.sh

```diff
@@ -20,8 +20,13 @@ cd dav1d
 mkdir build
 cd build
 
-# This only works on linux.
-android_bin="${1}/toolchains/llvm/prebuilt/linux-x86_64/bin"
+# This only works on linux and mac.
+if [ "$(uname)" == "Darwin" ]; then
+  HOST_TAG="darwin"
+else
+  HOST_TAG="linux"
+fi
```

Comment:
Optional: If Bash has a lowercase function, we can just set `HOST_TAG` to the lowercase of `$(uname)`.

Reply:
Ack. I've seen some systems report "Linux<with_some_suffix>". So this takes care of that.